gobblin-service/src/main/java/org/apache/gobblin/service/modules/utils/FlowCompilationValidationHelper.java

```diff
@@ -59,6 +69,30 @@ public final class FlowCompilationValidationHelper {
   private final FlowStatusGenerator flowStatusGenerator;
   private final boolean isFlowConcurrencyEnabled;
 
+  @Inject
+  public FlowCompilationValidationHelper(Config config, SharedFlowMetricsSingleton sharedFlowMetricsSingleton,
+      UserQuotaManager userQuotaManager, FlowStatusGenerator flowStatusGenerator) {
+    try {
+      String specCompilerClassName = ServiceConfigKeys.DEFAULT_GOBBLIN_SERVICE_FLOWCOMPILER_CLASS;
+      if (config.hasPath(ServiceConfigKeys.GOBBLIN_SERVICE_FLOWCOMPILER_CLASS_KEY)) {
+        specCompilerClassName = config.getString(ServiceConfigKeys.GOBBLIN_SERVICE_FLOWCOMPILER_CLASS_KEY);
```

Comment:
isn't there a method in `ConfigUtils` (or similar utility) that we could leverage to provide such fallback to a default?

Reply:
I do not think so.

Reply:
what about this one:
```
String ConfigUtils::getString(Config config, String path, String def)
```
?

Reply:
oh i misread your ask.